pypeit/core/telluric.py

```diff
@@ -653,9 +659,9 @@ def general_spec_reader(specfile, ret_flam=False):
     meta_spec = dict(bonus=bonus)
     meta_spec['core'] = spect_dict
 
-    return wave, counts, counts_ivar, counts_gpm, meta_spec, head
+    return wave, wave_grid_mid, counts, counts_ivar, counts_gpm, meta_spec, head
 
-def save_coadd1d_tofits(outfile, wave, flux, ivar, gpm, spectrograph=None, telluric=None,
+def save_coadd1d_tofits(outfile, wave, flux, ivar, gpm, wave_grid_mid=None, spectrograph=None, telluric=None,
```

Comment:
Update the docs for this new keyword parameter.

Reply:
Done.